sx-interaction.el

```diff
@@ -278,7 +278,12 @@ likes."
        (sx-question-get-from-comment .site_par .comment_id) 'focus)
       (sx--find-in-buffer 'comment .comment_id))
      (.notification_type
-      (sx-message "Viewing notifications is not yet implemented"))
+;     (sx-message "Viewing notifications is not yet implemented"))
+      (let ((start 0))
+       (while (string-match "\\(http[^\"]+\\)\"" .body start)
```

Comment:
Why is this wrapped in a loop? Do we want to get the last link in `body`? What links come before it?

Reply:
My observation of the notifications in my own inbox was that when there is more than one link, the first is to a general item, such as a question or answer, and the second is to an action url, such as a page to review someone's edit to your question/answer.

Reply:
Seems like we should just start the search from the bottom, then.